src/styles/components/_toc.scss

```diff
@@ -0,0 +1,154 @@
+@import '../../styles/settings/colors';
+@import '../../styles/settings/type';
+@import '../../styles/tools/breakpoints';
+@import '../../styles/tools/mixins';
+
+@mixin TOC_LINKS {
+  @include bp(md) {
+    color: $PRIMARY_TEXT_COLOR;
+
+    &.w-toc__active--link,
+    &:hover {
+      color: $WEB_PRIMARY_COLOR;
+    }
+  }
+}
+
+.w-toc__active--border {
+  border-left: 3px solid $WEB_PRIMARY_COLOR;
+  margin-left: -24px;
+  padding-left: 21px;
+}
+
+div#content.w-toc-open {
+  > * {
+    @include bp(md) {
+      width: calc(100% - 268px);
+
+      .w-actions {
+        right: calc(24px + 268px);
+      }
+    }
+
+    .w-toc__button--wrapper {
+      display: none;
+    }
+
+    web-table-of-contents {
+      display: block;
+    }
+  }
+}
+
+web-table-of-contents {
+  background-color: $WHITE;
+  border-bottom: 1px solid $BORDER_COLOR;
+  border-top: 1px solid $BORDER_COLOR;
+  display: none;
+  padding: 24px;
```

Comment:
I think putting padding on the element in this way will prevent you from being able to do the little blue rectangle to the left of the scrollSpy'd anchor.

![image](https://user-images.githubusercontent.com/1066253/83907906-08b11500-a71b-11ea-926d-1da84e0744a2.png)